doc/classes/RenderingServer.xml

```diff
@@ -2536,6 +2543,15 @@
 			<description>
 			</description>
 		</method>
+		<method name="mesh_surface_update_index_region">
+			<return type="void" />
+			<param index="0" name="mesh" type="RID" />
+			<param index="1" name="surface" type="int" />
+			<param index="2" name="offset" type="int" />
+			<param index="3" name="data" type="PackedByteArray" />
+			<description>
+			</description>
```

Comment:
Needs a description. I know the other methods don't have descriptions yet, but we really need to ensure everything is documented when added